helpers/helpers.go

```diff
@@ -153,6 +156,70 @@ func GetIncludedBundles(filename string) ([]string, error) {
 	return includes, nil
 }
 
+// UnpackFile unpacks a .tar or .tar.gz/.tgz file to a given directory
+// Should be roughly equivalent to "tar -x[z]f file -C dest"
```

Comment:
Document what is the expected behavior when the files being unpacked already exist.